src/main/java/com/startup/models/Teacher.java

```diff
@@ -0,0 +1,91 @@
+package com.startup.models;
+
+public class Teacher {
+
+    private long teacherId;
+    private String firstName;
+    private String lastName;
+    private String gender;
+    private int cnp;
+    private String address;
```

Comment:
address can be changed and divided to a more specific address type? For example "3425 Stone Street, Apt. 2A, Jacksonville, FL 39404." So why not use that as an example but conformed to a Romanian address, for example, streetNumber, streetName, apartmentNumber, cityName, county, zipCode?

Reply:
we can get this type of street from frontend through dto ( I mean in the part of front, in POST request we can have all these labels( address -- streetNumber, streetName, apartmentNumber etc. ), and after that we can join all these dates in 1 variable string, because in database I think it is reasonable to save address only in 1 column ? 
what do you think about this idea ?

Reply:
Yes, that is totally fine. I will alter the student model too with only one string address data type.